### DISCUSSION TITLE
Wagtailmenus

### DISCUSSION THREAD

**Discussion by DBonbon** [wagtail/wagtail-localize#479] [General]
I noted that I cannot run wagtail-localize with wagtailmenus. 

Once I comment:
 'wagtailmenus.context_processors.wagtailmenus',

...my code functions just fine. 

In case it could help, I thought to share this comment. 

I don't propose that it should be done though.

+'Chapeau' for this elegant project.

**Comment by zerolab** [comment]
Hey @DBonbon, would you be able to add a traceback for the error? It is possible this may be an issue for wagtailmenus, but worth having as much detail as possible anyway. Thank you.
